src/python/clawutil/data.py

```diff
@@ -462,6 +462,10 @@ def __init__(self, pkg, num_dim):
             self.add_data(amrclaw.AmrclawInputData(self.clawdata),'amrdata')
             self.add_data(amrclaw.RegionData(num_dim=num_dim),'regiondata')
             self.add_data(amrclaw.GaugeData(num_dim=num_dim),'gaugedata')
+            try:
+                self.add_data(amrclaw.AdjointData(num_dim=num_dim),'adjointdata')
+            except:
+                pass # for backward compatibility
```

Comment:
Is the try/except really necessary?  Any released version should have this in the future.

Reply:
It won't be necessary in the long run but we are still working on the amrclaw changes needed to support this.  But maybe it would be best to merge the current version of this AdjointData object first into amrclaw so this never throws an exception.  So hold off on merging this yet.